src/OpenTelemetry/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   function when configuring a view (applies to individual metrics).
   ([#5542](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5542))
 
+* Fixed a race condition for MetricPoint reclaim scenario which could have led
+  to a measurement being dropped.
+  ([#5546](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5546))
```

Comment:
I'm worried the impact isn't super clear and we might cause some panic. Could we somehow indicate it is an opt-in thing? Like...

* Fixed a race condition for MetricPoint reclaim scenario (enabled via `OTEL_DOTNET_EXPERIMENTAL_METRICS_RECLAIM_UNUSED_METRIC_POINTS`) which could have led
  to a measurement being dropped.
  ([#5546](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5546))

Reply:
Updated.